GameObject.cpp

```diff
@@ -1,41 +1,40 @@
+#include <sstream>
 #include "Header.h"
 #include "Board.h"
 
-inline void error(const std::string message)
+inline void error(const std::string &message)
 {
     std::cout << "RUN_TIME_ERROR : " << message << std::endl;
 }
 
-int toint(char c){
+inline void log(const std::string &message) {
+    std::cout << "LOG : " << message << std::endl;
+}
```

Comment:
nowa funkcja log, bo nie zawsze chcemy wołać error a trzeba skrócić to używanie couta rozwinięte